### PR TITLE
BUG/documentation: setting wrappers for specific binaries do not work

### PR DESCRIPTION
**Pull request by ruda**
When wrapping process, the parameter for a local wrap is defined by using a shell-like glob interface (`\*make` for example), the documentation is actually referring to the old behavior, regarding the use of absolute and relative paths, so it will fails if the user tries to execute.

This PR updates the documentation in synchronization with the current logic of `--wrapper` option.

See card: https://trello.com/c/rTPoo3JT/412-bug-setting-wrappers-for-specific-binaries-do-not-work

### CODE REVIEW

File: man/avocado.rst [hunk 1]
```diff
@@ -378,41 +378,45 @@ WRAP PROCESS IN TESTS
 =====================
 
 Avocado allows the instrumentation of applications being
-run by a test in a transparent way.
+run by a test in a transparent way. The user specify a script
```
Comment (clebergnu) [inline man/avocado.rst]: There is actually a typo here: "The user specifies..."

**Comment by lmr** [comment]
I'm glad to see this was a documentation problem, rather than a code problem :)

Apart from the little typo fixes suggested by @clebergnu, I'm fine with this.

**Comment by clebergnu** [comment]
I'm glad that the documentation is being fixed. There was some extra things I mentioned on the original card, so, I opened another way so that we can deal with that later:

https://trello.com/c/hwem7R1D/423-gdb-and-wrapper-parameter-style-inconsistent

File: man/avocado.rst (hunk 1 above)
Comment (ruda) [inline man/avocado.rst]: OK, fixing this.

**Comment by ruda** [comment]
Closing this in favor of #682

**Pull request closed without merging**